src/qililab/digital/circuit_transpiler.py

```diff
@@ -138,9 +144,9 @@ def route_circuit(
         coupling_map: list[tuple[int, int]] | None = None,
         iterations: int = 10,
     ) -> tuple[Circuit, dict[str, int]]:
-        """Routes the virtual/logical qubits of a circuit, to the chip's physical qubits.
-
+        """Routes the virtual/logical qubits of a circuit, to the chip's physical qubits. And returns/logs the final layout of the qubits.
 
+        If passed, this is done, with the passed `placer`, `router` and `routing_iterations` params (if not, default ones are applied).
```

Comment:
```suggestion
        """Routes the virtual or logical qubits of a circuit to the physical qubits of a chip. Returns and/or logs the final qubit layout. 

This process uses the provided `placer`, `router`, and `routing_iterations` parameters if they are passed; otherwise, default values are applied.
```